libbpf-cargo/src/gen.rs

```diff
@@ -303,20 +303,27 @@ fn gen_skel_prog_defs(
     object: *mut libbpf_sys::bpf_object,
     obj_name: &str,
     open: bool,
+    mutable: bool,
 ) -> Result<()> {
     if ProgIter::new(object).next().is_none() {
         return Ok(());
     }
```

Comment:
I should probably add something like this to `gen_skel_map_defs`. I'll do it now.